tools_docker/Debian_New_Uwsgi/Dockerfile

```diff
@@ -34,6 +34,7 @@ RUN apt-get install -qy $(cat "${PACKAGE_DIR}/run_backend_packages.list") \
 # - installs optional dependencies
 ####
 FROM base AS build_backend
+ENV SETUPTOOLS_USE_DISTUTILS stdlib
```

Comment:
missing `=` no ?

Reply:
You're right… I fixed it.

Reply:
In fact both work (https://docs.docker.com/engine/reference/builder/#env, "Alternative syntax"), but as other `ENV` variables are written with `=` it is a better idea to keep the same syntax everywhere

Reply:
ok. thanks for this information \o/